mlocal/checks/project-post.chk

```diff
@@ -45,6 +45,12 @@ config_add_def NVIDIALIBS_FILE SINGULARITY_CONFDIR \"/nvliblist.conf\"
 config_add_def SESSIONDIR LOCALSTATEDIR \"/singularity/mnt/session\"
 config_add_def SINGULARITY_SUID_INSTALL $with_suid
 config_add_def PLUGIN_ROOTDIR LIBEXECDIR \"/singularity/plugin\"
+config_add_def ENGINE_CONFIG_ENV \"C\"
+config_add_def ENGINE_CONFIG_CHUNK_ENV \"CC\"
```

Comment:
Not really a fan of this naming, as `CC=` in the envirionment has another common meaning, even if this is something not exposed to the user.

We lose a few bytes of space, but wouldn't `SINGULARITY_CONFIG` and `SINGULARITY_CONFIG_CHUNKS` be much more obvious for people who might end up tracing things?